package.json

```diff
@@ -85,9 +86,8 @@
     ],
     "loader": [
       "ts-node/esm",
-      "./build/coffee-esm.mjs",
       "./build/hera-esm.mjs",
-      "./dist/esm.mjs"
+      "./node_modules/@danielx/civet/dist/esm.mjs"
```

Comment:
This is the version that will parse the source and test files. Once parsed the tests are run against `main.civet`, `lib.civet`, `parser.hera`, etc.

Reply:
Ah, got it. (If only we could add a comment to a JSON file. 😉)

That seems fine, because we don't generally use fancy features in the test infrastructure, only inside the triple quotes.  Actually, in some ways, it will be an improvement; I've run into issues where I break the Civet compiler (and ran `yarn build`) and then `yarn test` crashes, with unhelpful error messages.  This should avoid that sort of issue.